### ISSUE TITLE
version unknown

### ISSUE THREAD

**Issue by sbonaretti** [rasbt/watermark#50]
Hi! Thanks for developing `watermark`, it is indeed fundamental to make my workflows reproducible! However, it seems like it cannot read the version of some packages - see attachment. Is there a way to read the version of these packages? Where does `watermark` get the version from? Thanks!

<img width="542" alt="ScreenShot" src="https://user-images.githubusercontent.com/13563701/60822857-8cc6e980-a1a6-11e9-8c35-039525ae2232.png">

**Comment by rasbt** [comment]
I think multiprocessing won't have a version number because it's already included in Python. Hence, it's version number would be whatever your Python's version is.

Regarding `SimpleITK` and `itk`, it looks like they don't support the common `__version__` attribute. Do you know by chance how you would get the version from within the package after importing it? It seems like there is nothing within that package that lists the version. Hm.

**Comment by sbonaretti** [comment]
At the end i solved my issue using:  
```
import pkg_resources
pkg_resources.get_distribution("SimpleITK").version
```
as suggested [here](https://stackoverflow.com/questions/3524168/how-do-i-get-a-python-modules-version-number-through-code). Not the cleanest way, but it works for now.

Thanks!

**Comment by rasbt** [comment]
That's good to know, thanks! This means we could add an exception (if-clause) for this particular case. This could actually be extended to all packages that don't have a `__version__` attribute -- very useful to know, thanks!

**Comment by rasbt** [comment]
Should be fixed now via #54
